src/NumericalAlgorithms/Interpolation/InterpolationTargetWedgeSectionTorus.hpp

```diff
@@ -0,0 +1,220 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include <algorithm>
+#include <pup.h>
+
+#include "DataStructures/DataBox/DataBox.hpp"
+#include "DataStructures/DataBox/DataBoxTag.hpp"
+#include "NumericalAlgorithms/Interpolation/SendPointsToInterpolator.hpp"
+#include "NumericalAlgorithms/Interpolation/Tags.hpp"
+#include "Parallel/ConstGlobalCache.hpp"
+#include "Utilities/TMPL.hpp"
+#include "Utilities/TaggedTuple.hpp"
+
+namespace intrp {
+
+namespace OptionHolders {
+/// \brief A solid torus of points, useful, e.g., when measuring data from an
+/// accretion disc.
+///
+/// The torus' cross section (e.g., a cut at \f$\phi=0\f$) is a wedge-like shape
+/// bounded by \f$r_{\text{min}} \le r \le r_{\text{max}}\f$ and
+/// \f$|\theta| \le \theta_{\text{half-angle}}\f$.
+///
+/// The grid points are located on surfaces of constant \f$r\f$, \f$\theta\f$,
+/// and \f$\phi\f$. There are `NumberOfRadialPoints` points uniformly spaced in
+/// \f$r\f$ between `MinRadius` and `MaxRadius` (including these endpoints);
+/// `NumberOfThetaPoints` points uniformly spaced in \f$\theta\f$ from
+/// `ThetaHalfAngle` below the equator to `ThetaHalfAngle` above the equator,
+/// (including these endpoints); and `NumberOfPhiPoints` uniformly distributed
+/// azimuthally.
+///
+/// \note At least two grid points are required per dimension.
+///
+/// \note Input coordinates (radii, angles) are interpreted in the frame given
+/// by the `Frame` template parameter of the `Actions::WedgeSectionTorus`.
+struct WedgeSectionTorus {
+  struct MinRadius {
+    using type = double;
+    static constexpr OptionString help = {"Inner radius of torus"};
+    static type lower_bound() { return 0.0; }
```

Comment:
I'm not sure how uniformly we have been doing this, but I believe the `bound` functions can all be `noexcept`

Reply:
Sure, I'll fix this one. More generally, very few of these `bound` functions are marked `noexcept`: I'm also happy to fix this if you assign an issue to me.